bbb/services.py

```diff
@@ -457,6 +463,20 @@ def _isAuthorized(self, buildername, scopes):
         else:
             return True
 
+    def _refreshAllowedBuilders(self):
+        now = arrow.now().timestamp
+        if self.allowed_builders is None or (now - self.allowed_builders_age) > 300:
+            try:
+                builders = requests.get(ALL_THE_THINGS_URL).json()['builders']
```

Comment:
I'm not sure what to do if this fails the first time. Ideally I think I'd like this to stop processing the message and try again later.